### ISSUE TITLE
Update contents documentation folder

### ISSUE THREAD

**Issue by mwakok** [SATAY-LL/Transposonmapper#49]
The documentation folder contains outdated documentation and needs to be reviewed. We could also add a pdf version of the jupyter-book as documentation, as well as a pdf version of the help.md file. We can add this documentation to the Docker image and resolve #45

**Comment by leilaicruz** [comment]
I have made a book of the Jupyter Book and a pdf of the help file , so we can copy them in the dockerfile to be visualize using the program you mentioned , will you try to do it?  https://github.com/SATAY-LL/Transposonmapper/tree/update_doc/documentation

**Comment by mwakok** [comment]
Yes, I will update both the GUI and the Dockerfile in this branch
